schemata_json_schema/src/Plugin/Type/TypeMapperPluginManager.php

```diff
@@ -34,7 +34,7 @@ class TypeMapperPluginManager extends DefaultPluginManager implements FallbackPl
    *   The module handler to invoke the alter hook with.
    */
   public function __construct(\Traversable $namespaces, ModuleHandlerInterface $module_handler) {
-    parent::__construct('Plugin/json_schema/type_mapper', $namespaces, $module_handler, 'Drupal\json_schema\Plugin\TypeMapperInterface', 'Drupal\json_schema\Annotation\TypeMapper');
+    parent::__construct('Plugin/type_mapper', $namespaces, $module_handler, 'Drupal\schemata_json_schema\Plugin\type_mapper\TypeMapperInterface', 'Drupal\schemata_json_schema\Annotation\TypeMapper');
```

Comment:
Shouldn't the plugin namespace be 'Plugin/schemata_json_schema/type_mapper'? Otherwise this is declaring no other module can have a type_mapper plugin in that location.